lib/tty/box.rb

```diff
@@ -341,7 +341,7 @@ def top_right_corner(border)
     #
     # @api private
     def top_titles_size(title)
-      title[:top_left].to_s.size + title[:top_center].to_s.size + title[:top_right].to_s.size
+      color().strip(title[:top_left].to_s).size + color().strip(title[:top_center].to_s).size + color().strip(title[:top_right].to_s).size
```

Comment:
I'd omit the brackets from `color()` and call method directly like `color.strip`. The line is also getting a bit long. How about turning it into multiple lines?